### PR TITLE
Bug 1756421: bring back explain tests

### PR DESCRIPTION
**Pull request by soltysh**
Supersedes https://github.com/openshift/origin/pull/23935
/assign @mfojtik

### CODE REVIEW

File: test/extended/cli/explain.go [hunk 1]
```diff
@@ -93,25 +102,46 @@ var (
...
-		{Group: "operator.openshift.io", Version: "v1", Resource: "kubeschedulers"},
-		{Group: "operator.openshift.io", Version: "v1", Resource: "networks"},
 		{Group: "operator.openshift.io", Version: "v1", Resource: "openshiftapiservers"},
-		{Group: "operator.openshift.io", Version: "v1", Resource: "openshiftcontrollermanagers"},
-		{Group: "operator.openshift.io", Version: "v1", Resource: "servicecas"},
-		{Group: "operator.openshift.io", Version: "v1", Resource: "servicecatalogapiservers"},
-		{Group: "operator.openshift.io", Version: "v1", Resource: "servicecatalogcontrollermanagers"},
+
+		// FIXME
+		// {Group: "operator.openshift.io", Version: "v1", Resource: "authentications"},
```
Comment (sttts) [inline test/extended/cli/explain.go]: fixed

Comment (stlaz) [reply test/extended/cli/explain.go]: https://github.com/openshift/origin/pull/24165

**Pull request merged**